sympy/solvers/ode/systems.py

```diff
@@ -1508,6 +1508,8 @@ def _higher_order_ode_solver(match):
     is_euler = is_transformed and type == "type1"
     is_higher_order_type2 = is_transformed and type == "type2" and 'P' in match
 
+    print(match.get('rhs', None))
```

Comment:
I hope this was for debugging(similarly other such statements)?

Reply:
Yes, I have removed these lines locally. Forgot to fix up these statements